src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -50,6 +51,18 @@ const TableHeroImage = ({
           role="presentation"
           className={styles.image}
         />
+        {leaverStatus !== undefined && leaverStatus > 1 &&
```

Comment:
I think leaverStatus > 1 is sufficient since `undefined > 1 === false`, but extra checks can't hurt